<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kol</class>
 <widget class="QDialog" name="kol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Касса</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>65</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Если взвешанное</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>кг.</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>44</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>шт.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Если упакованно</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Код</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
